.travis.yml

```diff
@@ -14,7 +14,7 @@ install:
   - pip install -e . --cache-dir $HOME/.pip-cache/
 script:
   - flake8 molo
-  - molo scaffold testapp --include=molo.yourtips ^yourtips/
+  - molo scaffold testapp --include=molo.yourtips ^yourtips/ --include=likes ^likes/
```

Comment:
I tested adding it to the yourtips urls, the issue is testing locally (py.test) and with travis. This was the only way I could get both to work with the url namespace.